### DISCUSSION TITLE
Anzeige PV Energie etc.

### DISCUSSION THREAD

**Discussion by mbrockeu** [brutella/hkknx-public#282] [Ideas]
Ich würde gerne meinen aktuellen Verbrauch bzw. Erzeugung der PV Anlage irgendwie in HomeKit bekommen.

Da ich jetzt nix passendes gefunden hatte wollte ich es über ein Helligkeitssensor machen, aber das funktioniert natürlich nicht.

Gibt es irgendeine Möglichkeit die Informationen als Sensor bereitzustellen und ggfs. in Automationen zu nutzen?

**Comment by brutella** [comment]
Leider gibt es keine Möglichkeit eigene Geräte, die nicht von HomeKit vordefiniert sind, zu erstellen. Technisch wäre es mit dem HomeKit-Protokoll möglich, aber die Apple Home-App würde es nicht anzeigen. 

Drittanbieter HomeKit-Apps, wie zB [Home+](https://itunes.apple.com/us/app/home-smart-home-automation/id995994352), können das aber. Trotzdem möchte ich nicht Funktionalität in hkknx einbauen, was eine andere App als die Apple Home-App voraussetzt.

**Comment by flobxy** [comment]
Hi @mbrockeu ,
ich lasse mir meine PV Daten via Skript in hkknx in eine InfluxDB schicken und visualisiere einige Werte dort. Ich spiele immer noch mit rum, aber es funktioniert bereits sehr gut!

<img width="1336" alt="Screenshot 2024-06-22 at 19 42 27" src="https://github.com/brutella/hkknx-public/assets/1721741/c1705c6c-7459-4b3d-bf59-d82bad666743">

**Reply by brutella** [reply]
Das ist sicher die flexibelste Lösung. Ich selbst speichere auch die Daten separate in einer InfluxDB.

Ich habe aber Ideen, eigene Messwerte in hkknx anzeigen zu lassen. In der neuen Version 3.0 werden unter Monitoring erstmal die aufgezeichneten Date von konfigurierten Geräten angezeigt. In einem nächsten Schritt, sollen auch Geräte-unabhängige Daten angezeigt werden können – zB PV-Daten.
